pkg/apb/rhcc_registry.go

```diff
@@ -19,5 +40,60 @@ func (r *RHCCRegistry) Init(config RegistryConfig, log *logging.Logger) error {
 // LoadSpecs - Load Red Hat Container Catalog specs
 func (r *RHCCRegistry) LoadSpecs() ([]*Spec, int, error) {
```

Comment:
Good call, updated this function along with a couple others to reflect that change.